mesheryctl/internal/cli/root/root.go

```diff
@@ -58,50 +60,71 @@ func (f *TerminalFormatter) Format(entry *log.Entry) ([]byte, error) {
 	return append([]byte(entry.Message), '\n'), nil
 }
 
-// rootCmd represents the base command when called without any subcommands
-var rootCmd = &cobra.Command{
+var (
+	availableSubcommands = []*cobra.Command{}
+)
+
+// RootCmd represents the base command when called without any subcommands
+var RootCmd = &cobra.Command{
 	Use:   "mesheryctl",
 	Short: "Meshery Command Line tool",
 	Long:  `Meshery is the service mesh management plane, providing lifecycle, performance, and configuration management of service meshes and their workloads.`,
+	//Args:  cobra.MinimumNArgs(1),
 	// Uncomment the following line if your bare application
 	// has an action associated with it:
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Is it a typo here or if not RunE sounds confusing?

Reply:
RunE is a cobra function that returns error. 
We need it for the following mistakes that a user can make 
```
return errors.New("sub-command not found : " + "\"" + args[0] + "\"")
```